applications/mne_scan/plugins/eegosports/eegosports.cpp

```diff
@@ -137,7 +137,7 @@ void EEGoSports::init()
     m_bWriteDriverDebugToFile = false;
     m_bCheckImpedances = false;
 
-    m_sElcFilePath = settings.value(QString("EEGOSPORTS/elcFilePath"), QString(QCoreApplication::applicationDirPath() + "../resources/general/3DLayouts/standard_waveguard64_duke.elc")).toString();
+    m_sElcFilePath = settings.value(QString("EEGOSPORTS/elcFilePath"), QString(QCoreApplication::applicationDirPath() + "./resources/general/3DLayouts/standard_waveguard66.elc")).toString();
```

Comment:
`./` is not needed since the absolute application path `QCoreApplication::applicationDirPath()` is prepended.